test/src/client_utils.dart

```diff
@@ -261,8 +262,9 @@ abstract class _Harness {
 
     final clientSubscription = fromClient.stream.listen(
         expectAsync1(handleServerMessage, count: serverHandlers.length),
-        onError: expectAsync1((_) {}, count: 0),
-        onDone: expectAsync0(doneHandler ?? () {}, count: expectDone ? 1 : 0));
+        onError: expectAsync1((dynamic _) {}, count: 0),
+        onDone: expectAsync0(doneHandler as void Function()? ?? () {},
```

Comment:
Instead of casting here, specify the type of the param to `runTest`.

Reply:
Done